docs/development-guide.md

```diff
@@ -122,13 +122,30 @@ Signed-off-by: Random J Developer <random@developer.example.org>
 The format can be described more formally as follows:
 
 ```text
-<subsystem>: <what changed>
+<component>: <subject of the change>
 <BLANK LINE>
-<why this change was made>
+<paragraph(s) with reason/description>
 <BLANK LINE>
-<footer>
+<signed-off-by>
 ```
 
+The `component` in the subject of the commit message can be one of the following:
+
+* `cephfs`: bugs or enhancements related to CephFS
+* `rbd`: bugs or enhancements related to RBD
+* `doc`: documentation updates
+* `util`: utilities shared between components use `cephfs` or `rbd` if the
+   change is only relevant for one of the type of storage
+* `journal`: any of the journalling functionalities
+* `helm`: deployment changes for the Helm charts
+* `build`: anything related to building Ceph-CSI, the executable or container
```

Comment:
let's add deploy or template to this one, it's good to go

Reply:
Added `deploy` with comment about the Kubernetes templates.